CHANGELOG.md

```diff
@@ -37,8 +37,12 @@ ChangeLog
 * #197: The `$children` property on components has been changed from `public`
   to `protected`. Use the `children()` method instead to get a flat list of
   objects.
+<<<<<<< HEAD
```

Comment:
Addressed in https://github.com/fruux/sabre-vobject/commit/87c6a50dd93506784ac70ac228e62ef93f0db262. Good.